### PR TITLE
refactor: use same pci sidebar for mobile and desktop

### PR DESCRIPTION
**Pull request by marie-j**
MANAGER-2996

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html [hunk 1]
```diff
@@ -89,3 +105,17 @@ <h2 class="mb-0 d-inline-block text-white" data-ng-bind="$ctrl.project.descripti
...
 </div>
+<div class="oui-navbar-backdrop"
+     aria-hidden="true"
+     tabindex="-1"
+     data-ng-class="{ 'oui-navbar-backdrop_active': $ctrl.isOpen }"
+     ng-click="$ctrl.close()"
+     role="button">
+</div>
+<button
+    class="position-fixed bg-transparent d-none sidebar-toggler"
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html]: +`type="button"`

File: packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html [hunk 2]
```diff
@@ -89,3 +105,18 @@ <h2 class="mb-0 d-inline-block text-white" data-ng-bind="$ctrl.project.descripti
     </div>
     <cloud-sidebar-project-list></cloud-sidebar-project-list>
 </div>
+<div class="oui-navbar-backdrop"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html]: `div` with `role="button"` that is `aria-hidden`  🤔

File: packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html [hunk 3]
```diff
@@ -1,26 +1,42 @@
...
+            <div class="d-flex" data-ng-if="!$ctrl.isLoading && $ctrl.project">
+                <div class="w-100 mr-3 minw-0">
+                    <div class="w-100 text-truncate">
+                        <h2 class="mb-0 d-inline-block font-weight-normal" data-ng-bind="$ctrl.project.description"></h2>
+                    </div>
+                    <oui-clipboard model="$ctrl.project.project_id"></oui-clipboard>
+                </div>
+                <div class="my-auto">
+                    <button type="button" class="oui-button oui-button_primary oui-button_small-width rounded-circle project-list-toggle" data-ng-click="$ctrl.toggleProjectsList()">
+                        <span class="oui-icon oui-icon-chevron-right text-white"></span>
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html]:
```suggestion
                        <span class="oui-icon oui-icon-chevron-right text-white" aria-hidden="true"></span>
```

File: packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html [hunk 4]
```diff
@@ -1,26 +1,42 @@
...
-                <button type="button" class="btn btn-primary rounded-circle" style="padding: 6px 13px; background-color: rgb(40, 89, 192);" data-ng-click="$ctrl.toggleProjectsList()">
-                    <span class="oui-icon oui-icon-chevron-right text-white"></span>
-                    <span class="sr-only" data-translate="cloud_sidebar_public_cloud_project_toggle"></span>
-                </button>
+            <div class="d-flex" data-ng-if="!$ctrl.isLoading && $ctrl.project">
+                <div class="w-100 mr-3 minw-0">
+                    <div class="w-100 text-truncate">
+                        <h2 class="mb-0 d-inline-block font-weight-normal" data-ng-bind="$ctrl.project.description"></h2>
+                    </div>
+                    <oui-clipboard model="$ctrl.project.project_id"></oui-clipboard>
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html]:
```suggestion
                    <oui-clipboard data-model="$ctrl.project.project_id"></oui-clipboard>
```

File: packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html [hunk 5]
```diff
@@ -1,26 +1,42 @@
...
         </div>
-        <div class="d-flex" data-ng-if="!$ctrl.isLoading && $ctrl.project">
-            <div class="w-100 mr-3 minw-0">
-                <div class="w-100 text-truncate">
-                    <h2 class="mb-0 d-inline-block font-weight-normal" data-ng-bind="$ctrl.project.description"></h2>
-                </div>
-                <oui-clipboard model="$ctrl.project.project_id"></oui-clipboard>
+        <div class="w-100">
+            <div data-ng-if="$ctrl.isLoading">
+                <oui-skeleton size="xs"></oui-skeleton>
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html]:
```suggestion
                <oui-skeleton data-size="xs"></oui-skeleton>
```

File: packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html [hunk 6]
```diff
@@ -1,26 +1,42 @@
...
+                <oui-skeleton class="mt-2" size="s"></oui-skeleton>
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html]:
```suggestion
                <oui-skeleton class="mt-2" data-size="s"></oui-skeleton>
```

**Pull request merged**